webpack/assets/javascripts/react_app/pages/AuditsPage/__snapshots__/AuditsPage.test.js.snap

```diff
@@ -5,30 +5,15 @@ exports[`AuditsPage rendering render audits page 1`] = `
   beforeToolbarComponent={null}
   breadcrumbOptions={null}
   customBreadcrumbs={null}
+  documentationURL="https://theforeman.org/manuals/1.24/index.html#4.1.4Auditing"
```

Comment:
Is this really the only way? This means we'll break the tests when we start the 1.25 release cycle. Perhaps a helper function that returns the shortversion so you can actually stub it out in tests?

Reply:
added a mock to the version